k8s/nginx/production/api-vs.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   host: api.airqo.net
   upstreams:
     - name: auth-service
-      service: airqo-stage-auth-api-svc
+      service: airqo-auth-api-svc
```

Comment:
`api.airqo.net` was pointing to staging instead of production

Reply:
thanks @NoahNsimbe